### PR TITLE
signup page component created with UI tests passing

### PR DESCRIPTION
**Pull request by Seyi-Toluhi**
We have written two tests for signup form UI to check that the input fields render and that when filled, the inputs become its values. We will include more tests for the actual form submission once the backend routes are set up.

### CODE REVIEW

**Review by F-Gulmez — DISMISSED** [review]

File: Frontend/src/pages/SignupPage.jsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+};
+
+const handleNeurodiversityChange = (event) => {
+setNeurodiversity(event.target.value);
+};
+
+
+return (
+<>
+    <h2>Signup</h2>
```
Comment (yulingyou) [inline Frontend/src/pages/SignupPage.jsx:55]: For better user experience and search engine optimisation, it is suggested to have a title tag on the signup page.

**Review by yulingyou — CHANGES_REQUESTED** [review]

File: Frontend/src/pages/SignupPage.jsx (hunk 1 above)
Comment (Seyi-Toluhi) [reply Frontend/src/pages/SignupPage.jsx:55]: ok i can get that sorted

Comment (yulingyou) [reply Frontend/src/pages/SignupPage.jsx:55]: Thank you!

Comment (Seyi-Toluhi) [reply Frontend/src/pages/SignupPage.jsx:55]: Done and i'll do same for login component

Comment (yulingyou) [reply Frontend/src/pages/SignupPage.jsx:55]: Hi Seyi, thanks! I just remembered that the title tag is supposed to be wrapped inside the `<head>` element. I'll double-check with the coach to see what their opinion is and let you know!

Comment (Seyi-Toluhi) [reply Frontend/src/pages/SignupPage.jsx:55]: yes Daisy, thats true for vanilla js. I am not sure how that works with React component as we've never had HTML headers in them. Would be good to know how to add headers to react components if possible.

Comment (Seyi-Toluhi) [reply Frontend/src/pages/SignupPage.jsx:55]: thats updated now daisy for this PR and login PR

Comment (yulingyou) [reply Frontend/src/pages/SignupPage.jsx:55]: The headers are currently in `Frontend/index.html`. Maybe for now, it's okay to hardcode the title. I've also added the suggestion to have a 'dynamic title' for our mirror board - proposed Features.

**Review by yulingyou — APPROVED** [review]

**Review by JackCollett — APPROVED** [review]

**Pull request merged**